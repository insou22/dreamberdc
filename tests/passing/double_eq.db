3.14 == "3.14"! //true