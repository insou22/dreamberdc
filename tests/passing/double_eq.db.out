Program {
    items: [
        Statement(
            Expr(
                ExpressionStatement {
                    expr: TwoEq(
                        TwoEqExpr {
                            lhs: NumericLit(
                                NumericLit {
                                    digits: "3.14",
                                },
                            ),
                            rhs: StringLit(
                                StringLit {
                                    contents: "3.14",
                                },
                            ),
                        },
                    ),
                    debug: false,
                },
            ),
        ),
    ],
}